features/unit/abijson.feature

```diff
@@ -0,0 +1,75 @@
+@unit.abijson
+@unit
+Feature: AbiJson
+  Scenario Outline: Serialize Method object from sig into json 
+    When I create the Method object from "<methodsig>"
+    And I serialize the Method object into json
+    Then the produced json should equal "<jsonfile>" loaded from "<directory>"
```

Comment:
I'd recommend also adding a final step to test decoding the response. Something like `And the deserialized Method object equals the original`, which would decode the JSON and compare it to the original Method object. 

Same suggestion for Interface and Contract.